glossary.asciidoc

```diff
@@ -198,7 +195,7 @@ funding transaction::
     From the perspective of the Bitcoin network, the process of opening a channel by creating a RSMC is started by creating the funding transaction and finished by broadcasting it to the Bitcoin network and have it included in the blockchain.
     The value of the funding transaction is exactly the capacity of the payment channel.
     The output of the funding transaction is a 2-of-2 multisignature script (multisig) where each channel partner controls one key.
-    It is supposed to be spent by the commitment transactions or by the closing transaction.
+    It is spent by the commitment transactions or by the closing transaction.
```

Comment:
I think the word supposed is correct. It can happen that it is never spent or that ownsers of the channel define something complete different.

Reply:
Ok.  Again, my technical knowledge is lacking.  However, "supposed to be" has the idea that something should happen, but it didn't.  E.g. "I was supposed to go to work today."

How about changing it to:
It may be spent by the commitment transactions or by the closing transaction.

or
It may or may not be spent by the commitment transactions or by the closing transaction.

Reply:
I think your initial suggestion is actually acurate enough. following the protocol it it will eventually be spent by either the commitment or closing tx. we could add the world `eventually` but I think that might be too much detail.